config.yml

```diff
@@ -74,8 +74,11 @@ ShapeFile: "/home/ubuntu/integrated_methane_inversion/resources/shapefiles/Permi
 ## Inversion
 ## Note PriorError is a relative percentage 
```

Comment:
Shouldn't call it a percentage but a fraction, I think. Since 0.5 doesn't mean 0.5%. Maybe something like this:

## Note PriorError is expressed as a fraction (e.g., 0.5 = 50%)